code/game/objects/structures/droppod.dm

```diff
@@ -209,7 +210,23 @@ GLOBAL_LIST_INIT(blocked_droppod_tiles, typecacheof(list(/turf/open/space/transi
 	reserved_area = SSmapping.RequestBlockReservation(3,3)
 
 	drop_state = DROPPOD_ACTIVE
+	GLOB.droppod_list -= src
 	update_icon()
+	flick("[icon_state]_closing", src)
+	addtimer(CALLBACK(src, PROC_REF(launch_pod), user), 2.5 SECONDS)
+
+///actually launches the pod
+/obj/structure/droppod/proc/launch_pod(mob/user)
+	if(!launch_allowed)
+		drop_state = DROPPOD_READY
+		GLOB.droppod_list += src
+		for(var/atom/movable/deployed AS in contents)
+			deployed.forceMove(loc)
+		update_icon()
+		if(user)
+			to_chat(user, span_notice("Error. Ship calibration unavailable. Please %#&ç:*"))
```

Comment:
why not ballon?

Reply:
Existing alerts are chat, so I left it for consistancy. I can update them all if you want though.